app/controllers/audits/allocations_controller.rb

```diff
@@ -6,6 +6,7 @@ class AllocationsController < BaseController
     decorates_assigned :content_items
 
     def index
+      @my_content_items_count = FindContent.my_content(current_user.uid).count
```

Comment:
I would prefer not to hide code via inheritance and be explicit in what instance variables set each controller. The code would be easier to read and maintain

Reply:
@pmanrubia if we don't want to use inheritance, then why do we even have `BaseController`? If this variable had lived in `BaseController`, then this bug wouldn't have happened.

Reply:
I would have to prevent the bug by covering it with tests. 
Regarding inheritance in `BaseController` it is being used to reuse methods that apply to all controllers. It is not hiding the magic of instance variables.

Reply:
Surely we should be consistent about "is a" vs "has a"? We could have commonised that code by creating eg a `FilterToParamConverter` class or similar, and then use that in the controllers, rather than lean on inheritance for common methods.

Reply:
Discussed assigning @my_content_items_count in the base controller with Pablo, he preferred explicitly assigning the variable in each controller for ease of comprehension.

Nice catch with regards to the reports page. Ty.

Reply:
Added tests to cover the bug 👍